src/main/java/Duke.java

```diff
@@ -1,10 +1,209 @@
+        String deadlineDate;
+
+
+        while (!startsWithBye) {
+
+            //waiting for user input
+            line = in.nextLine();
+
+            //checking command
+            startsWithBye = line.startsWith("bye");
+            startsWithList = line.startsWith("list");
+            startsWithDone = line.startsWith("done");
+            startsWithDeadline = line.startsWith("deadline");
+            startsWithTodo = line.startsWith("todo");
+            startsWithEvent = line.startsWith("event");
+            startsWithDelete = line.startsWith("delete");
+
+            try {
+
+                if (startsWithList) { //list command given
+
+                    System.out.println("Here are the tasks in your list:");
+                    for (numeration = 0; numeration < itemNumber; numeration++) {
+                        System.out.println(numeration + 1 + "." + taskings.get(numeration).toString());
+                    }
+
+                } else if (startsWithDone) { //done command given
+                    itemDone = Integer.parseInt(line.substring(5));
+//                    list[itemDone].markAsDone();
+                    taskings.get((itemDone - 1)).markAsDone();
+                    System.out.println("Nice! I've marked this task as done:\n    \u2713 " + taskings.get(itemDone - 1).getDescription());
+//                    System.out.println("Nice! I've marked this task as done:\n    \u2713 " + list[itemDone].getDescription());
+
+                } else if (startsWithDeadline) { //deadline command given
+                    indexOfBackslash = line.indexOf("/");
+                    deadlineDate = line.substring(indexOfBackslash + 4);
+                    line = line.substring(9, indexOfBackslash - 1);
+//                    list[itemNumber] = new Deadline(line, deadlineDate, isDone);
+//                    System.out.println("Got it. I've added this task:\n  " + list[itemNumber].toString() + "\nNow you have " + itemNumber + " tasks in the list.");
+                    taskings.add(new Deadline(line, deadlineDate, isDone));
+                    System.out.println("Got it. I've added this task:\n  " + taskings.get(itemNumber).toString() + "\nNow you have " + (itemNumber + 1) + " tasks in the list.");
+                    itemNumber++;
+
+                } else if (startsWithTodo) { //todo command given
+//                    list[itemNumber] = new Todo(line.substring(5), isDone);
+                    taskings.add(new Todo(line.substring(5), isDone));
+                    System.out.println("Got it. I've added this task:\n  " + taskings.get(itemNumber).toString() + "\nNow you have " + (itemNumber + 1 )+ " tasks in the list.");
+                    itemNumber++;
+
+                } else if (startsWithEvent) { //event command given
+                    indexOfBackslash = line.indexOf("/");
+                    deadlineDate = line.substring(indexOfBackslash + 4);
+                    line = line.substring(6, indexOfBackslash - 1);
```

Comment:
Perhaps you could store these magic numbers as a variable so the intention behind subtracting / adding them is easily understood